test/k8s-local-cluster-test/provision-cluster

```diff
@@ -85,7 +85,7 @@ for dep in "${deps[@]}"; do
     if [ ! -x "$path_to_executable" ]; then
         echoerr "You are required to have $dep installed on your system..."
         echoerr "Please install $dep and try again. "
-        exit -1
+        exit 3
```

Comment:
y tho

Reply:
was using bash exit codes improperly, apparently exit only accepts positive integers and exit 0, 1, 2 and supposed to be special-ish, so starting custom error codes at 3 :)